thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -416,9 +417,11 @@ public void uninstallPush(@NonNull String installationID) throws ThingIFExceptio
      * of the command by calling {@link #getCommand}.
      * @throws ThingIFException Thrown when failed to connect IoT Cloud Server.
      * @throws ThingIFRestException Thrown when server returns error response.
+     * @deprecated use {@link ThingIFAPI#postNewCommand(CommandForm)}.
```

Comment:
Lets' not deprecated this method.
Still useful when Title/ Metadata/ Description is not required.